src/Cli/Monolog/ConsoleHandler.php

```diff
@@ -95,7 +101,10 @@ public function close(): void
         parent::close();
     }
 
-    protected function write(array $record): void
+    /**
+     * @param LogRecord|array $record
+     */
+    protected function write($record): void
```

Comment:
```suggestion
    protected function write(LogRecord|array $record): void
```
PHP Versions >= 8 support union types.